src/components/Table/TableRow.tsx

```diff
@@ -43,6 +43,13 @@ export const TableRow = ({
       onChange({ selectedValue: rowData });
     }
   };
+  const isSelected = () => {
+    if (JSON.stringify(rowData) === JSON.stringify(selectedValue)) {
+      return true;
+    } else {
+      return false;
+    }
+  };
```

Comment:
Making it a function makes it run just as many times as before.. :wink: 
```suggestion
  const isSelected = JSON.stringify(rowData) === JSON.stringify(selectedValue);
```
This would also require rewriting the calls below to just reference the variable instead.